src/lib/manager/aws/lambda-manager.ts

```diff
@@ -114,6 +114,9 @@ export class LambdaManager {
         timeout: props.timeoutInSecs ? cdk.Duration.seconds(props.timeoutInSecs) : cdk.Duration.minutes(1),
         vpc: vpc,
         vpcSubnets: vpcSubnets,
+        tracingConfig: {
```

Comment:
Better to include the full object `tracingConfig: props.tracingConfig`... to give more flexibility to override other configs in the calling construct